content/en/docs/concepts/configuration/scheduler-perf-tuning.md

```diff
@@ -45,41 +51,37 @@ algorithmSource:
 percentageOfNodesToScore: 50
 ```
 
-{{< note >}}
-In clusters with zero or less than 50 feasible nodes, the
-scheduler still checks all the nodes, simply because there are not enough
-feasible nodes to stop the scheduler's search early.
-{{< /note >}}
+{{< note >}} In clusters with less than 50 feasible nodes, the scheduler still
+checks all the nodes, simply because there are not enough feasible nodes to stop
+the scheduler's search early. {{< /note >}}
 
 **To disable this feature**, you can set `percentageOfNodesToScore` to 100.
 
 ### Tuning percentageOfNodesToScore
 
-`percentageOfNodesToScore` must be a value between 1 and 100
-with the default value of 50. There is also a hardcoded minimum value of 50
-nodes which is applied internally. The scheduler tries to find at
-least 50 nodes regardless of the value of `percentageOfNodesToScore`. This means
-that changing this option to lower values in clusters with several hundred nodes
-will not have much impact on the number of feasible nodes that the scheduler
-tries to find. This is intentional as this option is unlikely to improve
-performance noticeably in smaller clusters. In large clusters with over a 1000
-nodes setting this value to lower numbers may show a noticeable performance
-improvement.
+`percentageOfNodesToScore` must be a value between 1 and 100 with the default
+value being calculated based on the cluster size. There is also a hardcoded
+minimum value of 50 nodes or 5% whichever is larger. This means that changing
```

Comment:
if 5% is the lower bound, then setting anything from 1 to 4 makes no sense, right?

Reply:
Good point. The 5% applies only when the option is not provided. I clarified it.